.github/ISSUE_TEMPLATE/falsepositive.yml

```diff
@@ -0,0 +1,112 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms
+name: False Positive Report
+description: File a Phishing report
+labels: ["False Positive"]
+assignees:
+  - mitchellkrogza
+  - funilrys
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We understand being listed on a Phishing Database like this can be frustrating and embarrassing for many web site owners.
+
+        The first step is to remain calm.
+        
+        The second step is to rest assured one of our maintainers will address your issue as soon as possible.
+
+        Please make sure you have provided as much information and details as possible to help speed up the process.
+
+  - type: textarea
+    id: record
+    attributes:
```

Comment:
@funilrys I'm open for suggestions, currently I just can't come up with any better, Maybe @mitchellkrogza or @iam-py-test can as it is there native language.

Reply:
Let's leave it like that for now.